ansible/roles/sshkey-host-access/tasks/main.yml

```diff
@@ -0,0 +1,73 @@
+---
+
+- name: ensure the existence of the user directory
+  file:
+    path: "{{ KEY_LOC_PATH }}"
+    owner: "{{ KEY_LOC_OWNER }}"
+    group: "{{ KEY_LOC_GROUP }}"
+    mode: 0700
+    state: directory
+  delegate_to: "{{ play_on }}"
```

Comment:
Okay, just a matter of preference here, I would rename play_on to target_host. shell doesn't not make me think of a host..